docs/ref/contrib/postgres/lookups.txt

```diff
@@ -48,6 +48,20 @@ The ``trigram_word_similar`` lookup can be used on
 
 .. _`pg_trgm extension`: https://www.postgresql.org/docs/current/pgtrgm.html
 
+``trigram_strict_word_similar``
+-------------------------------
```

Comment:
A new paragraph is unnecessary and it's inconsistent with the current structure of this doc.